src/ucp/core/ucp_worker.h

```diff
@@ -221,6 +223,9 @@ typedef struct ucp_worker_mpool_key {
 KHASH_TYPE(ucp_worker_mpool_hash, ucp_worker_mpool_key_t, ucs_mpool_t);
 typedef khash_t(ucp_worker_mpool_hash) ucp_worker_mpool_hash_t;
 
+/* EP configurations storage */
+UCS_ARRAY_DEFINE_INLINE(ep_config_arr, unsigned, ucp_ep_config_t);
```

Comment:
maybe the inline function should be in ucp_worker.inl
here only do UCS_ARRAY_DECLARE_TYPE